src/BuildScriptGenerator/Php/PhpLanguageDetector.cs

```diff
@@ -12,7 +12,7 @@
 
 namespace Microsoft.Oryx.BuildScriptGenerator.Php
 {
-    internal class PhpLanguageDetector : ILanguageDetector
+    internal class PhpLanguageDetector : IPlatformDetector
```

Comment:
Maybe change its name to  `PhpPlatformDetector.cs` as well since here we renamed all `Language` to `Platform`. It'll look better if all conforms. Same as other platforms: `NodePlatformDetector.cs`, `PythonPlatformDetector.cs` and `DotnetCorePlatformDetector.cs`.

Reply:
And it will match well with PhpPlatformInstaller.

Reply:
Good catch. Will do